api/envoy/config/filter/http/lua/v2/lua.proto

```diff
@@ -18,4 +20,23 @@ message Lua {
   // be properly escaped. YAML configuration may be easier to read since YAML supports multi-line
   // strings so complex scripts can be easily expressed inline in the configuration.
   string inline_code = 1 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
From an API standpoint, it's kind of strange that we still require inline code if we now support named code. Should we now potentially relax requiring inline_code and deprecate it, and optionally allow a root/global code by name?